packages/sync-actions/src/product-types-actions.js

```diff
@@ -256,11 +256,56 @@ const generateUpdateActionsForAttributeEnumValues = (
   ]
 }
 
-export const actionsMapForHints = (nestedValuesChanges = {}) => [
-  ...generateUpdateActionsForAttributeDefinitions(
-    nestedValuesChanges.attributeDefinitions
-  ),
-  ...generateUpdateActionsForAttributeEnumValues(
-    nestedValuesChanges.attributeEnumValues
-  ),
-]
+const generateChangeAttributeOrderAction = (
+  attrsOld = [],
+  attrsNew = [],
+  updateActions = []
+) => {
+  if (!attrsOld.length || !attrsNew.length) return null
+
+  const orderNew = attrsNew.map(attribute => attribute.name)
```

Comment:
Let's try to find a better naming.. One could easily get confuse this with `Order` resource type

Reply:
Updated here 04415f14